pom.xml

```
@@ -24,8 +24,8 @@
     </licenses>
     
     <properties>
-        <jenkins.version>2.222</jenkins.version>
-        <revision>1.30</revision>
+        <jenkins.version>2.332.4</jenkins.version>
+        <revision>1.38</revision>
```

So sorry that my poor eyesight made this mistake.  I didn't have my glasses on and read the 1.30 as a 1.38.  Needs to be 1.30...
```suggestion
        <revision>1.30</revision>
```